data/mods/DinoMod/egg.json

```
@@ -1,4 +1,4 @@
-[
+
```

That was a wrong move.
```suggestion
[
```